packages/lodestar-cli/src/cmds/account/cmds/wallet/recover.ts

```diff
@@ -0,0 +1,59 @@
+import * as fs from "fs";
+import {ICliCommand} from "../../../../util";
+import {IGlobalArgs} from "../../../../options";
+import inquirer from "inquirer";
+import {createWalletFromArgsAndMnemonic, printUuidData} from "./utils";
+import {IWalletCreateArgs, walletCreateOptions} from "./create";
+
+export type IWalletRecoverArgs = IWalletCreateArgs & {
+  mnemonicInputPath: string;
+};
+
+export type ReturnType = string[];
+
+export const recover: ICliCommand<IWalletRecoverArgs, IGlobalArgs, ReturnType> = {
+  command: "recover",
+
+  describe: "Recovers an EIP-2386 wallet from a given a BIP-39 mnemonic phrase.",
+
+  examples: [
+    {
+      command: "account wallet recover",
+      description: "Recover wallet",
+    },
+  ],
+
+  options: {
+    ...walletCreateOptions,
+    mnemonicInputPath: {
+      description: "If present, the mnemonic will be read in from this file.",
+      type: "string",
+    },
+  },
+
+  handler: async (args) => {
+    const {mnemonicInputPath} = args;
+    let mnemonic;
+
+    console.log("\nWARNING: KEY RECOVERY CAN LEAD TO DUPLICATING VALIDATORS KEYS, WHICH CAN LEAD TO SLASHING.\n");
+
+    if (mnemonicInputPath) {
+      mnemonic = await fs.promises.readFile(mnemonicInputPath, "utf8");
+    } else {
+      const input = await inquirer.prompt([
```

Comment:
Do other clients have an option to input the mnemonic manually? If it leaves a trace in bash history this is security risk and we should only offer reading from a file.

Reply:
lighthouse does it :shrug:

Reply:
but yeah i can see the potential for security risk there.  we could potentially change the inquirer prompt to be "password" type instead of "input" like how we do for importing a validator

Reply:
If Lighthouse does it then I'm cool with it :+1: